app/models/season/phase_switcher.rb

```diff
@@ -4,6 +4,16 @@ def self.season
     Season.current
   end
 
+  def self.switcher(phase)
+    @phases = {
+        'Proposals' => :fake_proposals_phase,
```

Comment:
I wouldn't. Right now, the controller just acts as a YOLO-kind of messenger. It just passed whatever down to whatever owns the actual logic. Let that handle the input validation/sanitation.